ol-keycloak/ol-spi/src/main/java/edu/mit/keycloak/util/OpenAttribute.java

```diff
@@ -0,0 +1,5 @@
+package edu.mit.keycloak.util;
+
+public class OpenAttribute {
```

Comment:
I would prepend "Ol" to this class name in order to more easily distinguish it against OOTB.